src/main/resources/static/react/App.tsx

```diff
@@ -8,6 +8,15 @@ import { useId } from 'react';
 
 export function App() {
 	const id = useId();
-	console.debug(toStr({id}));
-	return (<div className='blue'>{dayjs().format()}</div>);
+	console.debug(`Hello from React inside a site page. React app id: ${toStr({id})}`);
+
+	return (
+		<>
+			<div className='blue'>{dayjs().format()}
+				<h3>This is a React app inside a site page.</h3>
```

Comment:
The string "site", needs to become a prop, so it can be "webapp" inside the webapp.

Reply:
Will fix in https://github.com/enonic/starter-tsup/issues/230